src/Microsoft.AspNetCore.SignalR.Core/HubOptions.cs

```diff
@@ -2,27 +2,16 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Collections.Generic;
 
 namespace Microsoft.AspNetCore.SignalR
 {
     public class HubOptions
     {
-        /// <summary>
-        /// The default keep-alive interval. This is set to exactly half of the default client timeout window,
-        /// to ensure a ping can arrive in time to satisfy the client timeout.
-        /// </summary>
-        public static readonly TimeSpan DefaultKeepAliveInterval = TimeSpan.FromSeconds(15);
+        public TimeSpan? NegotiateTimeout { get; set; } = null;
 
-        public TimeSpan NegotiateTimeout { get; set; } = TimeSpan.FromSeconds(5);
+        public TimeSpan? KeepAliveInterval { get; set; } = null;
```

Comment:
Bring the docs back, and add a note about `null` indicating the default interval (15 seconds).

Reply:
The comment is a bit more complicated than that because `null` could indicate the default, or it could indicate that the global option should be used

Reply:
Ugh, yeah. Ok, save it for now, we'll need to doc it at some point though.